app/Http/Controllers/V4DB/AnimeController.php

```diff
@@ -172,64 +173,50 @@ public function main(Request $request, int $id)
             ->where('mal_id', $id)
             ->get();
 
-        if (
-            $results->isEmpty()
-            || $this->isExpired($request, $results)
-        ) {
+        if ($results->isEmpty()) {
             $response = Anime::scrape($id);
 
             if (HttpHelper::hasError($response)) {
                 return HttpResponse::notFound($request);
             }
 
-            if ($results->isEmpty()) {
-                $meta = [
-                    'createdAt' => new UTCDateTime(),
-                    'modifiedAt' => new UTCDateTime(),
-                    'request_hash' => $this->fingerprint
-                ];
-            }
-            $meta['modifiedAt'] = new UTCDateTime();
-
-            $response = $meta + $response;
-
-            if ($results->isEmpty()) {
-                Anime::query()
-                    ->insert($response);
-            }
-
-            if ($this->isExpired($request, $results)) {
-                Anime::query()
-                    ->where('mal_id', $id)
-                    ->update($response);
-            }
-
-            $results = Anime::query()
-                ->where('mal_id', $id)
-                ->get();
+            Anime::query()
+                ->insert(
```

Comment:
afaik if you run the insert this way, it won't trigger laravel scout's search index update process.

Reply:
@pushrbx  That's interesting to know. Should I do it the way you've done it in https://github.com/jikan-me/jikan-rest/pull/227 or take care of it during merge conflict?

Reply:
We can add the change to that PR then.